src/pybind/mgr/dashboard/requirements.txt

```diff
@@ -9,3 +9,5 @@ requests
 Routes
 six
 -e ../../../python-common
+prettytable
+pyyaml
```

Comment:
@liewegas @sebastian-philipp I think you missed to add those dependencies in the ceph.spec.in file

```console
failed: [mon2 -> mon0] (item=dashboard) => changed=true 
  ansible_loop_var: item
  cmd:
  - ceph
  - --cluster
  - ceph
  - mgr
  - module
  - enable
  - dashboard
  delta: '0:00:00.433621'
  end: '2020-02-18 14:00:06.796205'
  item: dashboard
  msg: non-zero return code
  rc: 2
  start: '2020-02-18 14:00:06.362584'
  stderr: 'Error ENOENT: module ''dashboard'' reports that it cannot run on the active manager daemon: No module named ''yaml'' (pass --force to force enablement)'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

Reply:
#33387